mujoco_ros_msgs/msg/EqualityConstraintParameters.msg

```diff
@@ -0,0 +1,11 @@
+string name
+mujoco_ros_msgs/EqualityConstraintType type
+mujoco_ros_msgs/SolverParameters solverParameters
+bool active
+string classParam
+string element1
+string element2
+float32 torquescale
+float32[] anchor
```

Comment:
Is there a case where the size of anchor is not 3 or can we replace `float32[]` with `geometry_msgs/Vector3`?